src/ModelFactory.php

```diff
@@ -74,6 +74,19 @@ final public static function createOne(array $attributes = []): Proxy
         return static::new()->create($attributes);
     }
 
+    /**
+     * A shortcut to create multiple models, based on a sequence, without states.
+     *
+     * @param iterable<array<string, mixed>>|callable(): iterable<array<string, mixed>> $sequence
+     *
+     * @return list<Proxy<TModel>&TModel>
```

Comment:
```suggestion
     * @return list<TModel&Proxy<TModel>>
```

Maybe reversing these will fix the cs-fixer issue/bug? `randomSet()` below has it like this with no issue.